examples/cloudml-sklearn-pipeline/requirements.txt

```diff
@@ -0,0 +1,8 @@
+matplotlib>=2.2.3
+seaborn>=0.9.0
+scikit-learn>=0.20.2
```

Comment:
I suppose you've checked all of those licenses and attributed them correctly?

Reply:
Sure 👍